src/linear_algebra/finite_dimensional.lean

```diff
@@ -330,6 +330,21 @@ iff.trans (by { rw ← findim_eq_dim, norm_cast }) (@dim_pos_iff_nontrivial K V
 lemma findim_pos [finite_dimensional K V] [h : nontrivial V] : 0 < findim K V :=
 findim_pos_iff.mpr h
 
+/-- A finite dimensional space has zero `findim` iff it is a subsingleton. -/
```

Comment:
Can you add a comment to each of these lemmas referencing the equivalent `dim K V = 0` lemma, since the names don't quite match?

Reply:
Would you think its better just to rename them to `findim_zero_iff_subsingleton` instead. Although I feel that `findim_eq_zero_iff_subsingleton` is more descriptive

Reply:
Frankly even if you did rename them, having the cross-links is handy.

But I think `subsingleton` is the result I would expect from `findim_zero_iff`, as any other result is a less convenient statement that follows trivially from projections of `subsingleton`.